### ISSUE TITLE
Script Fails when output for CLI is not JSON

### ISSUE THREAD

**Issue by msbishop** [awslabs/serverless-photo-recognition#12]
Found this out after some fumbling, I was set to text and that broke the SED/AWK etc. in the shell scripts. 

Easy to deal with, but would be great to fix by forcing JSON output for CLI commands independently.

**Comment by vbudilov** [comment]
Good point...I might even make this a requirement or modify the script to force it...let me take a look.
